### PR TITLE
SDK-246 [Python SDK] Add search_keys filter support

### PR DESCRIPTION
**Pull request by RommeroInCountry**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-246](https://incountry.atlassian.net/browse/SDK-246)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -350,6 +353,17 @@ key3={"$not": "value1"} # records with key3 not equal "value1"
 key4={"$not": ["value1", "value2"]} # records with key4 equal to neither "value1" or "value2"
 ```
 
+You can use special `search_keys` filter to search records by partial match (similar to `LIKE` SQL operator) amond record's text fields `key1, ..., key10`.
```
Comment (artemgolozubov) [inline README.md:356]:
```suggestion
You can use special `search_keys` filter to search records by partial match (similar to `LIKE` SQL operator) among record's text fields `key1, ..., key10`.
```

**Review by igor-alekseev-dev — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -66,6 +66,9 @@ You can turn off encryption (not recommended). Set `encrypt` property to `false`
     "endpoint_mask": str,       # Defines API base hostname part to use.
                                 # If set, all requests will be sent to https://${country}${endpointMask} host
                                 # instead of the default one (https://${country}-mt-01.api.incountry.io)
+
+    "hash_search_keys": bool    # tells the SDK not to hash search keys key1, ..., key10
+                                # which enables partial match search using search_keys find filter operator
```
Comment (artemgolozubov) [inline README.md:71]: IMO this comment is more clear in Node SDK

**Review by artemgolozubov — COMMENTED** [review]
Pls fix a typo

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by igor-alekseev-dev — APPROVED** [review]

**Pull request merged**